lib/Doctrine/DBAL/Query/QueryBuilder.php

```diff
@@ -999,7 +1072,11 @@ public function orHaving($having)
      */
     public function orderBy(string $sort, ?string $order = null)
     {
-        return $this->add('orderBy', $sort . ' ' . (! $order ? 'ASC' : $order), false);
+        $this->orderBy = [$sort . ' ' . (! $order ? 'ASC' : $order)];
```

Comment:
Let's remove the default `ASC`. It's implied by the SQL semantics of `ORDER BY` and doesn't need to be explicitly specified. Additionally, please use `if ($order !== null)` instead of `!$order` for better code coverage and type clarity.

Reply:
Done in 2aac6882b1349eb7eb3cf33be8c6b8a4518b10ad.